documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -1201,6 +1201,31 @@ NATS has a built-in distributed persistence system called https://docs.nats.io/n
 | memory
 | Controls how the messages are saved in the stream. Can be memory or file.
 
+|[[nats-jetstream-auth-jwt]]<<nats-jetstream-auth-jwt, `debezium.sink.nats-jetstream.auth.jwt`>>
+|
```

Comment:
```suggestion
|No default value
```
To avoid ambiguity, when a property has no default value, explicitly set the value of the **Default** column to `No default value`.